src/LiveQuery/QueryTools.js

```diff
@@ -90,6 +90,30 @@ function queryHash(query) {
 }
 
 /**
+ * contains -- Determines whether a constraint in the form of a list
```

Comment:
how about:

Determines if an object is contained in a list with special handling for Parse pointers.

Reply:
LOVE it!